{"name": "test_invalid_sign_in_get_request", "status": "passed", "steps": [{"name": "Fail sign in get request", "status": "passed", "start": 1648813615557, "stop": 1648813615806}], "start": 1648813615556, "stop": 1648813615806, "uuid": "35aaefe4-9b8a-4868-8e5a-95f7e411c266", "historyId": "b76b91d68b162f208ab544833704df5e", "testCaseId": "c03d7c083b451bb970c52430f0e4a590", "fullName": "tests.api_testcases.login_test.login_test.TestUserLogin#test_invalid_sign_in_get_request", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.login_test"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "149566-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.login_test.login_test"}]}